mozfun/norm/fenix_app_info/udf.sql

```diff
@@ -0,0 +1,50 @@
+CREATE OR REPLACE FUNCTION norm.fenix_app_info(app_id STRING, app_build_id STRING)
+RETURNS STRUCT<app_name STRING, channel STRING, app_id STRING> AS (
+  CASE
+  -- Note that order is important here; we move from more specific names to more general
+  -- so that we can properly ignore dataset suffixes like _stable or _live or _derived.
+  WHEN
+    app_id LIKE 'org_mozilla_fennec_aurora%'
+  THEN
+    STRUCT('Fenix', 'nightly', 'org.mozilla.fennec_aurora')
+  WHEN
+    app_id LIKE 'org_mozilla_firefox_beta%'
+  THEN
+    STRUCT('Fenix', 'beta', 'org.mozilla.firefox_beta')
+  WHEN
+    app_id LIKE 'org_mozilla_firefox%'
+  THEN
+    STRUCT('Fenix', 'release', 'org.mozilla.firefox')
+  WHEN
+    app_id LIKE 'org_mozilla_fenix_nightly%'
+  THEN
+    STRUCT('Firefox Preview', 'nightly', 'org.mozilla.fenix.nightly')
+  WHEN
+    app_id LIKE 'org_mozilla_fenix%'
+  THEN
+    IF(
+      -- See udf.fenix_build_to_datetime for info on build_id format;
+      -- the build_id constant here corresponds to 2020-07-03
+      app_build_id < '21850000',
```

Comment:
I took out the call to `udf.fenix_build_to_datetime` both for performance and to avoid needing a refactor of `udf.fenix_build_to_datetime` within the scope of this PR.